.github/workflows/docker.yml

```
@@ -62,25 +66,95 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Build and push
-        id: docker_build
+      - name: Build and push by digest
+        id: build
         uses: docker/build-push-action@v6
         with:
           context: ./
           file: ./Dockerfile
+          platforms: ${{ matrix.build.PLATFORM }}
           builder: ${{ steps.buildx.outputs.name }}
-          push: ${{ github.event_name != 'pull_request' }}
-          tags: ${{ steps.meta.outputs.tags }}
-          sbom: true
-          provenance: true
           labels: ${{ steps.meta.outputs.labels }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+          outputs: type=image,name=orhunp/git-cliff,name=ghcr.io/${{ github.repository_owner }}/git-cliff/git-cliff,push-by-digest=${{ github.event_name != 'pull_request' }},name-canonical=true
+          sbom: true
+          provenance: true
+
+      - name: Export digest
+        run: |
+          digest="${{ steps.build.outputs.digest }}"
+          mkdir -p /tmp/digests
+          touch "/tmp/digests/${digest#sha256:}"
+          cat "/tmp/digests/${digest#sha256:}"
+
+      - name: Upload digest
+        uses: actions/upload-artifact@v4
+        with:
+          name: digests-${{ env.PLATFORM_PAIR }}
+          path: /tmp/digests/*
+          if-no-files-found: error
+          retention-days: 1
+
+  merge:
+    name: Merge
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request'
+    needs:
+      - build
+    steps:
+      - name: Download digests
+        uses: actions/download-artifact@v4
+        with:
+          path: /tmp/digests
+          pattern: digests-*
+          merge-multiple: true
+
```

```suggestion
          merge-multiple: true

      - name: Set up Docker Buildx
          uses: docker/setup-buildx-action@v3
```

Wow sorry... i just noticed i suggested to remove the wrong lines. The docs mentioned in the description have this step as well, so i guess it is needed for the Docker meta step as well.

This will not solve the issue with the missing docker socket on MacOs though...

---

I think it's pretty clear ...
https://github.com/docker/setup-buildx-action/issues/176

---

But still doesn't matter as macos on the M1 chip cannot do docker builds anyway.